src/riak_kv_index_hashtree.erl

```diff
@@ -296,11 +302,12 @@ handle_call({delete, Items}, _From, State) ->
 handle_call(get_trees, _From, #state{trees=Trees}=State) ->
     {reply, Trees, State};
 
-handle_call({update_tree, Id}, From, State) ->
+handle_call({update_tree, Id, Callback}, From, State) ->
```

Comment:
@fadushin sure, makes sense for the API call. +1. think the diff just made it more confusing. you have the right call.